client/styles/spice.scss

```diff
@@ -0,0 +1,7 @@
+.mockup {
+  color:gray;
+  padding-top:1em;
+  width:100%;
+  text-align: center;
+  font-size: larger;
+}
```

Comment:
Not a huge deal, but maybe consistent indentation between HTML, CSS, and JSON? Or maybe there are conventions I'm unaware of. And of similar unimportance, git seems to prefer a newline at the end of the file.